lab-isaac/src/main.js

```diff
@@ -0,0 +1,41 @@
+import Cowsay from 'cowsay-browser';
+// import {SQUIRREL} from 'cowsay-browser';
+import Faker from 'faker';
+import './styles/main.scss';
+import React from 'react';
+import ReactDom from 'react-dom';
+
+class App extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      content: '',
+    };
+    this.handleClick = this.handleClick.bind(this);
+  }
+
+  handleClick(e) {
+    this.setState(prevState => {
+      return {content: Cowsay.say({text:`${Faker.lorem.words(10)}`})};
+    });
+  }
+
+  render() {
+    return (
+      <div className="application">
+        <h1>Generate Cowsay lorem</h1>
+        <button onClick={this.handleClick}>click me</button>
+        <pre>{this.state.content}</pre>
+      </div>
+    );
+  }
+}
+
+ReactDom.render(<App />, document.getElementById('root'));
+//Should have a property called content
```

Comment:
unnecessary comments

tip- if you're going to copy and paste the directions from the lab README (which is fine), copy them into your own README and slightly tweak them to be your apps README, describing the app